CodaProtocol/coda
# coda crash report

The zip file`coda_crash_report_<date-time>.tar.xz` in the config directory consists of 

    1. coda_short.log: ~4MB of coda.log
    2. registered_mask.dot
    3. frontier.dot
    4. coda_status.json
    5. crash_summary.json": Basic system infomation from `uname`, the daemon command that was run with, exception
    6. daemon.json

Manually tested it by crashing the daemon

Example error message:

```
2019-08-07 18:20:12 UTC [Info] No slots won in this epoch. Waiting for next epoch to check 
again, @1565202240000
2019-08-07 18:20:12 UTC [Info] Daemon ready. Clients can now connect
2019-08-07 18:20:12 UTC [Info] Generating crash report


  ☠  Coda Daemon crashed. The Coda Protocol developers would like to know why!

  Please:
    Open an issue:
      https://github.com/CodaProtocol/coda/issues/new

    Briefly describe what you were doing and attach the crash report /Users/deepthi/.coda-config/coda_crash_report_2019-08-07_18-20-12.206385.tar.xz
```

Note: Trust db and ledger db will be included in another PR, probably next release?

Partially fixes #3045
